doc/source/conf.py

```diff
@@ -215,3 +215,7 @@
 # List of patterns, relative to source directory, that match files and
 # directories to ignore when looking for source files.
 exclude_patterns = ['substitutions.rst', ]
+
+# numpydoc_show_class_members = False
+numpydoc_class_members_toctree = False
```

Comment:
built the docs locally, all looks good. for future reference, do you remember why this was necessary?

Reply:
I had forgotten, but without it I get dozens of warnings related to the FswtResult class.

See:. https://github.com/numpy/numpydoc/pull/6

Reply:
I had forgotten, but without it I get dozens of warnings related to the FswtResult class.

See:. https://github.com/numpy/numpydoc/pull/6

Reply:
Ah yes, now I remember. Thanks